src/config/section/compute.js

```diff
@@ -89,7 +89,18 @@ export default {
           dataView: true,
           groupAction: true,
           show: (record) => { return ['Stopped'].includes(record.state) },
-          args: (record, store) => { return ['Admin'].includes(store.userInfo.roletype) ? ['podid', 'clusterid', 'hostid'] : [] },
+          args: (record, store) => {
+            var fields = []
+            if (store.userInfo.roletype === 'Admin') {
+              fields = ['podid', 'clusterid', 'hostid']
+            }
+            if (record.hypervisor === 'VMware') {
+              if (store.apis.startVirtualMachine.params.filter(x => x.name === 'bootintosetup').length > 0) {
```

Comment:
@DaanHoogland I've tested and fixed a case here - we want to make Primate work with all recent/pass/future version of CloudStack; so if there's a feature adding new params for autogenerated forms, we can check if the param is available by checking the API discovery /cc @shwstppr @davidjumani